src/test_solvers/main.cpp

```diff
@@ -0,0 +1,32 @@
+#include "solvers.h"
+int main(){
+
+    // Initialize triplets for the matrix
+    std::vector<int> rowInd;
+    std::vector<int> colInd;
```

Comment:
Given that you want to give a good example here that is easy to understand, there is probably a benefit in not abbreviating all the variables. Consider descriptive names that make it immediate clear to someone looking at the code.

`rowInd` -> `rowIndex`, `val` -> `values` etc.

Reply:
b is the common name of the right-hand-side vector. I added a comment for it.